CMakeFiles/3.10.2/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,73 @@
+set(CMAKE_C_COMPILER "/usr/bin/cc")
```

Comment:
Szerintem ezeket a generált fájlokat kár a repoban tartani, elegendőek azok, amikből a program előállítható.